apps/docs/src/app/core/component-docs/tabs/examples/index.ts

```diff
@@ -15,7 +15,7 @@ import { TabSelectionExampleComponent } from './tab-selection-example/tab-select
 import { TabsExampleComponent } from './tabs-example/tabs-example-component';
 import { TabCollapsibleExampleComponent } from './tab-collapsible-example/tab-collapsible-example.component';
 import { TabCollapsibleOverflowExampleComponent } from './tab-collapsible-overflow-example/tab-collapsible-overflow-example.component';
-
+import { TabFormElementExampleComponent } from './tab-form-element-example/tab-form-element-example.component';
 export * from './tab-collapsible-overflow-example/tab-collapsible-overflow-example.component';
```

Comment:
leave a space of line between export and import statement

Reply:
modified.